### ISSUE TITLE
compatibility with R 4.0

### ISSUE THREAD

**Issue by ErvanCH** [hunzikp/velox#46]
After having updated R to version 4.0, it seems that velox is not available. Any chance to upgrade the package? Thanks

**Comment by HughSt** [comment]
Looks like it works OK if you install from GitHub, e.g.

```
library(devtools)
install_github("hunzikp/velox")
```

**Comment by dominiqueweber** [comment]
Yeah, luckily it still works.It is a fantastic package. Will it not be developed further?

**Comment by mokajee** [comment]
I am also trying to install this package on V4.0. I keep getting the following error: 

Error: Failed to install 'velox' from GitHub:
  (converted from warning) installation of package ‘C:/Users/KJXMO/AppData/Local/Temp/Rtmp888zFR/file217c6aed4d26/velox_0.2.0.9002.tar.gz’ had non-zero exit status

Any ideas on how to resolve this?
